File: Palindrome_Number.py
"""
Given an integer x, return true if x is a palindrome, and false otherwise.
"""
def isPalindrome(x):
    # convert integer to string and split it into a list of characters
    digits = list(str(x))
     # reverse the list and join it back into a string
    reverse = ''.join(digits[::-1])
    
     # compare the original string with the reversed string
    return str(x) == reverse

print(isPalindrome(-101)) #output: false
print(isPalindrome(101)) #output: true

File: Palindrome_Number.js
/* 
Given an integer x, return true if x is a palindrome, and false otherwise.
*/

function isPalindrome(x) {
  // convert integer to string and split it into an array
  const digits = String(x).split('');

  // reverse the array and join it back into a string
  const reversed = digits.reverse().join('');

  // compare the original string with the reversed string
  return String(x) === reversed;
}

console.log(isPalindrome(101))//Output: true
console.log(isPalindrome(21))//Output: false
